packages/contracts/utils/helpers.ts

```diff
@@ -1,130 +1,115 @@
 }
 
-export function getPluginInfo(networkName: string): any {
-  let pluginInfoFilePath: string;
-  let pluginInfo: any = {};
+export function getAragonDeploymentsInfo(networkName: string): any {
+  let aragonDeploymentsInfoFilePath: string;
+  let aragonDeploymentsInfo: any = {};
 
   if (['localhost', 'hardhat', 'coverage'].includes(networkName)) {
-    pluginInfoFilePath = '../../plugin-info-testing.json';
+    aragonDeploymentsInfoFilePath = '../../local-network-deployments.json';
   } else {
-    pluginInfoFilePath = '../../plugin-info.json';
+    aragonDeploymentsInfoFilePath = '../../production-network-deployments.json';
   }
 
   if (
-    existsSync(pluginInfoFilePath) &&
-    statSync(pluginInfoFilePath).size !== 0
+    existsSync(aragonDeploymentsInfoFilePath) &&
+    statSync(aragonDeploymentsInfoFilePath).size !== 0
   ) {
-    pluginInfo = JSON.parse(readFileSync(pluginInfoFilePath, 'utf-8'));
+    aragonDeploymentsInfo = JSON.parse(
+      readFileSync(aragonDeploymentsInfoFilePath, 'utf-8')
+    );
 
-    if (!pluginInfo[networkName]) {
-      pluginInfo[networkName] = {};
+    if (!aragonDeploymentsInfo[networkName]) {
+      aragonDeploymentsInfo[networkName] = {};
     }
   } else {
-    pluginInfo[networkName] = {};
+    aragonDeploymentsInfo[networkName] = {};
   }
-  return pluginInfo;
+  return aragonDeploymentsInfo;
 }
 
-function storePluginInfo(networkName: string, pluginInfo: any) {
+export function copyAragonDeploymentsInfoFromProdToLocal(
+  hre: HardhatRuntimeEnvironment
+) {
+  const productionNetworkName = getProductionNetworkName(hre);
+
+  const objToStore: any = {};
+  objToStore[hre.network.name] = getAragonDeploymentsInfo(
+    productionNetworkName
+  )[productionNetworkName];
+
+  writeFileSync(
+    '../../local-network-deployments.json',
+    JSON.stringify(objToStore, null, 2) + '\n'
+  );
+}
+
+function storePluginInfo(networkName: string, aragonDeploymentsInfo: any) {
   if (['localhost', 'hardhat', 'coverage'].includes(networkName)) {
     writeFileSync(
-      '../../plugin-info-testing.json',
-      JSON.stringify(pluginInfo, null, 2) + '\n'
+      '../../local-network-deployments.json',
+      JSON.stringify(aragonDeploymentsInfo, null, 2) + '\n'
     );
   } else {
     writeFileSync(
-      '../../plugin-info.json',
-      JSON.stringify(pluginInfo, null, 2) + '\n'
+      '../../production-network-deployments.json',
+      JSON.stringify(aragonDeploymentsInfo, null, 2) + '\n'
     );
   }
 }
 
+export function clearPluginInfo(networkName: string) {
+  if (['localhost', 'hardhat', 'coverage'].includes(networkName)) {
+    writeFileSync(
+      '../../local-network-deployments.json',
+      JSON.stringify({}, null, 2) + '\n'
+    );
+  } else {
+    throw "Information will be lost. You must delete 'production-network-deployments.json' manually.";
+  }
+}
+
 export function addDeployedRepo(
   networkName: string,
   repoName: string,
   contractAddr: string,
-  args: [],
+  args: any[],
```

Comment:
can we type this better? `unknown[]` is usually better than `any[]`

Reply:
Nice! 0299d35aaf908e089bb649c15fd4fdd6df42ab4a

I will try to get rid of these ugly helper functions, storing things locally.

Reply:
I got rid of them now